### PR TITLE
Retain original sc.ini instead of generating a new one

### PR DESCRIPTION
**Pull request by MoonlightSentinel**
Only append the required `LIB` path s.t. the remaining values match the expected defaults from dmd. Required because the generated config contains an invalid `LINKCMD` with the new default for 32 bit windows.

Also changes the `tools` build to use host compiler to avoid issues w.r.t. the OMF => MsCoff default transition (as LDC defaults to MsCoff for 32).

---

Extracted from #494

### CODE REVIEW

**Review by thewilsonator — APPROVED** [review]

**Comment by thewilsonator** [comment]
Please merge this when you're happy with it.

**Comment by MoonlightSentinel** [comment]
@thewilsonator Done, the windows build is passing again. Don't have write access to this repo though.

**Pull request merged**
